web/source/osk/oskManager.ts

```diff
@@ -285,21 +285,21 @@ namespace com.keyman.osk {
           Ldiv.className='kmw-osk-static';
           Ldiv.innerHTML = Lhelp;
           this._Box.appendChild(Ldiv);
-          if(activeKeyboard['KHF']) {
-            activeKeyboard['KHF'](this._Box);
+          if(activeKeyboard.scriptObject['KHF']) {
```

Comment:
The ['KH'] and ['KHF'] properties aren't very obviously defined, even to me, so I held off on creating long-name properties for them.

Reply:
`KH` = keyboard help text ([reference](https://help.keyman.com/developer/language/reference/kmw_helptext)): HTML help text which is a one liner intended for the status bar of the desktop OSK originally.
`KHF` = keyboard help file ([reference](https://help.keyman.com/developer/language/reference/kmw_helpfile)): This html stored inline replaces the OSK when present. e.g. see European Latin keyboards.